### PR TITLE
[UX] Improve cache information from chart dropdown on Dashboard page

### LINKED ISSUE #13332: [UX] Unnecessary cache information in chart dropdown on Dashboard page
### Description
The chart dropdown on the Dashboard page currently displays unnecessary cache information. This could be confusing or overwhelming for users who do not need this level of detail. This issue can be resolved by removing or hiding this cache information.

### Impact
Addressing this issue would streamline the user interface and make the Dashboard page more user-friendly. It would reduce clutter and potential confusion, leading to a more efficient and satisfying user experience. This would be particularly beneficial for users who prefer a simpler, more straightforward interface.

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

### PR DESCRIPTION
**Pull request by andersonjeccel**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [X]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #13332  

#### Description:
The chart dropdown on the Dashboard page displays technical cache/data information, we can make it better

The idea is to replace things like `data loaded in 3.3 ms from cache` by:
- Showing previously stored data (cache)
- Just retrieved latest data (database)

image

image

to be more user-friendly

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to Dashboard
3. Open any chart card dropdown
4. You will see only the cache info

### CODE REVIEW

**Comment by RCheesley** [comment]
I'm not sure that this would be a helpful change for users because it is important to know if your data has been cached and might not be showing the most up to date data? Otherwise you might make incorrect interpretations?

**Comment by andersonjeccel** [comment]
> I'm not sure that this would be a helpful change for users because it is important to know if your data has been cached and might not be showing the most up to date data? Otherwise you might make incorrect interpretations?

Cache is refreshed every time you refresh/reload the page

**Comment by escopecz** [comment]
I'm also not a fan as I find that information helpful. But I was thinking yesterday that it could be improved when I saw this on the Jira dashboard widgets:
Screenshot 2024-02-07 at 14 42 18
Seeing how old is the widget data is probably more useful than from which source it was loaded.

**Comment by andersonjeccel** [comment]
> I'm also not a fan as I find that information helpful. But I was thinking yesterday that it could be improved when I saw this on the Jira dashboard widgets: Screenshot 2024-02-07 at 14 42 18 Seeing how old is the widget data is probably more useful than from which source it was loaded.

This is a better idea, do you know the code I could place there?

**Comment by escopecz** [comment]
It won't be an easy change, I'm afraid.

**Comment by andersonjeccel** [comment]
> It won't be an easy change, I'm afraid.

I wouldn't have knowledge to develop a timer since last refresh inside of Mautic. Do I close this PR?

**Comment by escopecz** [comment]
Yeah, let's close this until we'll have time to improve the staleness of the widget data.

**Comment by andersonjeccel** [comment]
Updated the entire PR to recreate the message with available resources instead of removing

**Comment by abhisekmazumdar** [comment]
Before:

image

After:

image


If this is what we intend to achieve next, then I feel it works now.

**Review by abhisekmazumdar — APPROVED** [review]

**Comment by RCheesley** [comment]
There's a conflict to resolve here, but I really like the improvements suggested. Something I did wonder as a user, is if I see the data has been cached, how do I refresh the data? Is there any way to link to some docs which explain what these two options mean and show how to refresh the data?

Once the conflict is resolved it should be an easy one to re-test and get merged.

**Comment by andersonjeccel** [comment]
@RCheesley Conflicts solved

Recreated the message to be simpler

Regardnig your questions, I also have no idea, but this PR is good to go

**Review by RCheesley — CHANGES_REQUESTED** [review]
Something odd seems to have happened - the three dots for all the widgets show on the left, and the info shows on the right:

screenshot-mautic ddev site-2024 08 30-17_35_11

**Review by RCheesley — DISMISSED** [review]
Sorry I realised that my browser was caching some old files and since so much has been merged recently it was making everything wonky. Hard-refreshed and all is good, I like the terminology as it's much easier for a user to understand!

**Comment by andersonjeccel** [comment]
@RCheesley Ctrl + Shift + R is a magical shortcut
It reloads the page clearing your cache (perhaps you'll use Command in Mac)

**Comment by RCheesley** [comment]
Yeah I didn't realise there was a caching issue until I noticed the left side and the top were weird, and the logo was showing white on white! 🤦🏻

File: app/bundles/DashboardBundle/Resources/views/Widget/detail.html.twig [hunk 1]
```diff
@@ -15,22 +24,10 @@
                             <i class="ri-edit-line"></i> Edit
                         </a>
                     </li>
-                    <li role="separator" class="divider"></li>
-                    <li  class="dropdown-header">
-                        {{ 'mautic.dashboard.widget.load.time'|trans({'%time%': (widget.loadTime * 1000)|round(1)}) }}
```
Comment (escopecz) [inline app/bundles/DashboardBundle/Resources/views/Widget/detail.html.twig:20]: Shouldn't we also remove the translation in the ini file?

Comment (andersonjeccel) [reply app/bundles/DashboardBundle/Resources/views/Widget/detail.html.twig:20]: Done!

**Pull request merged**